lib_device_control/host/control_host_support.h

```diff
@@ -8,7 +8,10 @@
 #include <string.h>
 #include <assert.h>
 #include "control.h"
-#include "control_transport.h"
+#include "control_shared.h"
```

Comment:
Control.h already includes control shared?

Reply:
Good point. I will remove that include.